frontend/pages/policies/constants.ts

```diff
@@ -126,7 +126,7 @@ export const DEFAULT_POLICIES: IPolicyNew[] = [
     query: "SELECT 1 FROM mdm WHERE enrolled='true';",
     name: "MDM enrolled (macOS)",
     description:
-      "Required: osquery deployed with Orbit, or manual installation of macadmins/osquery-extension. Checks that a Mac is enrolled to MDM. Add a AND on identity_certificate_uuid to check for a specific MDM.",
+      "Required: osquery deployed with Fleet's agent (fleetd), or manual installation of macadmins/osquery-extension. Checks that a Mac is enrolled to MDM. Add a AND on identity_certificate_uuid to check for a specific MDM.",
```

Comment:
```suggestion
      "Checks that a Mac is enrolled to MDM. Add a AND on identity_certificate_uuid to check for a specific MDM.",
```

I think we can cut the first part. Deploying fleetd is best practice.